.github/workflows/aws-deployment.yml

```diff
@@ -0,0 +1,95 @@
+---
+    name: AWS Deployment
```

Comment:
Indentation seems off... this should be left-justified, and two spaces per indentation.

Reply:
There was some double-indenting on the source I began with.  I'm going to fix it in vs code 

https://raw.githubusercontent.com/allen-cell-animated/nucmorph-colorizer/main/.github/workflows/aws-deploy.yml